shared/constants/chat2/index.js

```diff
@@ -227,6 +227,20 @@ export const anyToConversationMembersType = (a: any): ?RPCChatTypes.Conversation
   }
 }
 
+const successfulInlinePaymentStatuses = ['completed', 'claimable']
+export const hasSuccessfulInlinePayments = (state: TypedState, message: Types.Message) => {
+  if (message.type !== 'text' || !message.inlinePaymentIDs) {
+    return false
+  }
+  for (let paymentID of message.inlinePaymentIDs) {
```

Comment:
i think this could be
`return message.inlinePaymentIDs.some(id => successfulInlinePaymentStatuses.includes(state.chat2.paymentStatusMap.get(id)?.status))
`